src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -109,6 +96,27 @@ sortObjects(Kokkos::View<unsigned int *, DeviceType> view)
   return bin_sort.get_permute_vector();
 }
 
+#if defined(KOKKOS_ENABLE_CUDA)
+// NOTE returns the permutation indices **and** sorts the morton codes
+inline Kokkos::View<size_t *, Kokkos::Device<Kokkos::Cuda, Kokkos::CudaSpace>>
```

Comment:
Let the compiler decide whether to inline or not that function.  Drop the `inline` unless you measured?

Reply:
In this version, the function was not a template, so `inline` is necessary for avoiding ODR problems.